recipe/conda_build_config.yaml

```diff
@@ -1,7 +1,3 @@
-cxx_compiler_version:          # [osx]
-  # need to downgrade on osx due to a bug that breaks the test suite
-  - 10                         # [osx]
```

Comment:
@isuruf

Is it possible to specify `cxx_compiler_version` for osx & osx-arm independently? I tried to keep clang 10 for osx but have clang 11 for osx-arm, and I did not find a way through various selectors & logical operators that would render correctly...

Reply:
`osx and x86_64`

Reply:
Thanks a lot! :)